007_scalarmultiplication.js

```diff
@@ -1,8 +1,8 @@
 let numbers = [1, 2, 3, 4, 5];
-const scalarMultiplication=(numbers)=>{
-for (let i = 0; i < numbers.length; i++) {
-   numbers[i] *=  2;
-  return numbers[i];
-}
-}
+const scalarMultiplication = (numbers) => {
+  for (let i = 0; i < numbers.length; i++) {
+    numbers[i] *= 2;
+    return numbers[i];

```

Comment:
Did you check the output?. Answer is wrong. Logic of this function is wrong. Cross check this and make it correct.